### PR TITLE
add devtron condition to devtools extension loader

### PR DESCRIPTION
**Pull request by jens-t**
followup on #234.

### CODE REVIEW

**Comment by jens-t** [comment]
Maybe it's better to create a wiki page and link to from the readme for the whole development section?

**Comment by hql287** [comment]
> Maybe it's better to create a wiki page and link to it from the readme for the whole development section?

Great idea! 👍

**Comment by hql287** [comment]
I'll try to write the first page about this tomorrow as i'm also writing the contributing guide.

**Comment by jens-t** [comment]
removed the readme changes ...

**Comment by hql287** [comment]
This is even better than a shortcut 👍

File: app.js [hunk 1]
```diff
@@ -154,6 +154,8 @@ function createPreviewWindow() {
 }
 
 function addDevToolsExtension() {
+  if (process.env.DEVTRON_DEV_TOOLS_PATH)
```
Comment (hql287) [inline app.js]: What would be the value of `DEVTRON_DEV_TOOLS_PATH` if I decided to use it?

Comment (jens-t) [reply app.js]: that would be the node_modules path, as it is in the devDependencies:
`DEVTRON_DEV_TOOLS_PATH=./node_modules/devtron`

Comment (hql287) [reply app.js]: It works! 😆

**Pull request merged**